### ISSUE TITLE
when validation_step/end not defined, val_loss still gets logged

### ISSUE THREAD

**Issue by awaelchli** [williamFalcon/pytorch-lightning#522]
**Describe the bug**
If the validation_step/end is not implemented by user, lightning still shows a mysterious validation loss in tqdm. Where does this come from?

**To Reproduce**
1. Take MNIST example in "basic_examples" folder (current master branch)
2. Uncomment `validation_step` and `validation_end` and run the gpu_template
3. Wait until epoch 2 starts
4. You will see:
`Epoch 2:  14%|█▎        | 150/1095 [00:10<01:05, 14.35batch/s, batch_nb=149, gpu=0, loss=0.177, train_loss=0.261, v_nb=1, val_acc=0.939, val_loss=0.19]`

**Expected behavior**
`val_acc` and `val_loss` should not be there. Where do these numbers come from?

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - Version: 0.5.3.2 (via pip install)

**Comment by awaelchli** [comment]
It could be related to #487.
